vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/IssuerMetadata.kt

```diff
@@ -21,7 +23,7 @@ data class IssuerMetadata(
      * OID4VCI: REQUIRED. The Credential Issuer's identifier.
      */
     @SerialName("credential_issuer")
-    val credentialIssuer: String,
+    val credentialIssuer: String? = null,
```

Comment:
Not trying to over-engineer stuff here, but would it not be nice to subclass for better nullability? (also applies to other parts of the codebase)

Reply:
As discussed offline, this is probably not that easy due to the complexity of possible features/extensions for OAuth